src/main/resources/com/google/api/codegen/nodejs/package.snip

```diff
@@ -1,23 +1,23 @@
 @snippet generate(metadata)
   {
-    "url": "{@metadata.url}",
+    "url": "{@metadata.homepage}",
     "name": "{@metadata.identifier}",
-    "version": "{@metadata.version}",
-    "author": "Google Inc.",
+    "version": "{@metadata.packageVersion.lower}",
+    "author": "{@metadata.author}",
     "description": "{@metadata.serviceName} client for Node.js",
     "main": "src/index.js",
     "files": [
       "src"
     ],
     "dependencies": {
-      "google-proto-files": "{@metadata.protoVersion}",
-      "google-gax": "{@metadata.gaxVersion}",
+      "google-proto-files": "{@metadata.protoVersion.lower}",
+      "google-gax": "{@metadata.gaxVersion.lower}",
       "extend": "^3.0.0"
     },
     "devDependencies": {
       "mocha": "3.2.0"
     },
-    "license": "Apache-2.0",
+    "license": "{@metadata.license}",
```

Comment:
The name `license` implies the content of the license; maybe this should be `licenseName`.